share/spice/get_gpscoordinates/get_gpscoordinates.js

```diff
@@ -0,0 +1,108 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_get_gpscoordinates = function(api_result){
+
+        if (api_result.error) {
```

Comment:
Also need to check if we get a vaild response at all eg.  `!api_result | api_result.error`